MapboxVision/src/main/java/com/mapbox/vision/telemetry/TotalBytesCounter.kt

```diff
@@ -12,45 +16,51 @@ internal interface TotalBytesCounter {
 
     fun millisToNextSession(): Long
 
+    fun fitInLimit(bytes: Long): Boolean
+
     class Impl(
-            private val sessionLengthMillis: Long = SESSION_LENGTH_MILLIS,
-            private val sessionMaxBytes: Long = SESSION_MAX_BYTES
+        private val sessionLengthMillis: Long = SESSION_LENGTH_MILLIS,
+        private val sessionMaxBytes: Long = SESSION_MAX_BYTES,
+        private val systemTime: SystemTime = SystemTime.Impl
     ) : TotalBytesCounter {
 
         companion object {
-            private const val SESSION_LENGTH_MILLIS = 60 * 60 * 1000L // one hour
+            private val SESSION_LENGTH_MILLIS by lazy(LazyThreadSafetyMode.NONE) { TimeUnit.HOURS.toMillis(1) }
```

Comment:
Do we really need lazy here?

Reply:
when we set this value through constructor we don't need this value.

Reply:
Right, but is it worth to create additional objects for this cases? I believe 1-time multiplication will be even faster then property access overhead

Reply:
yes, but I'd lite to use `TimeUnit` util class for time. It useful and clean. btw `SESSION_LENGTH_MILLIS ` doesn't call many times

Reply:
So can we go with `private val ... = TimeUnit.HOURS.toMillis(1)`?

Reply:
what are pros of this change?

Reply:
let's discuss it offline :)